docs/guides/VTEX-Platform-Overview/store-architecture/store-architecture.md

```diff
@@ -136,28 +138,50 @@ This model is easier to set up, integrate, and maintain compared to the [multi-a
 
 ### Multi-account
 
-In this architecture, a main account acts as a [seller](https://help.vtex.com/en/tutorial/what-is-a-seller--5FkLvhZ3Few4CWWIuYOK2w) in secondary accounts that act as [marketplaces](https://help.vtex.com/en/tutorial/configuring-the-marketplace-between-vtex-stores--tutorials_6520).
+In a multi-account architecture, a brand operates multiple VTEX accounts (typically one per country or market), allowing each to be independently localized and operated. This architecture is recommended for operations across multiple countries that require configuring different languages and currencies, where different teams manage each localized store.
```

Comment:
```suggestion
In a multi-account architecture, a brand operates multiple VTEX accounts, typically one per country or market, so that each can be localized and managed independently. This architecture is recommended for businesses operating in multiple countries that need to configure different languages and currencies, with different teams managing each localized store.
```